### ISSUE TITLE
RCS1264 `roslynator_use_var = never` Task<T> should be applied

### ISSUE THREAD

**Issue by nemu69** [dotnet/roslynator#1602]
**Product and Version Used**:

NuGet package v. 4.12.4

**Steps to Reproduce**:

in `.editorconfig`
```
roslynator_use_var = never
```
Code:
```csharp
internal class Demo
{
  public static void TaskDemo()
  {
    var toto = await Task.FromResult(string.Empty); // <--- Should trigger an error here
  }
}
```
**Actual Behavior**:
No error is raised.

**Expected Behavior**:
Suggest RCS1264.

**Temporary solution**
Use `dotnet_diagnostic.rcs1008.severity = error`
